### DISCUSSION TITLE
給高中生的建議

### DISCUSSION THREAD

**Discussion by lai1008** [1101-GS4719-C/Python-Community-GS4719-C#2241] [General]
想問一下大家有沒有甚麼給高中生的建議，因為在高中時對大學完全沒有想法，所以希望大家可以給學弟妹們一些自己對大學的看法或是在高中學習上的建議。

**Comment by Boying12** [comment]
在高中時期，建議多參加各種課外活動和社團，這不僅能幫助你發現自己的興趣所在，還能提升你的社交能力和團隊合作精神。這些經驗對於未來選擇大學科系和職業方向都會有很大幫助。

**Comment by Chattenoire** [comment]
多多讀課外的、有意義的書，提高自己思考的框架；此外大概就是透過社群/社團培養自己的額外專長吧。
至於社交，如果本來就在行，就依個人選擇看要不要花時間搞社團；若是不擅長，就多花時間練習/觀察，畢竟這個能力太重要。

**Comment by christaaah** [comment]
我覺得高中最重要的除了讀書之外就是培養一點團隊合作的意識了。到了大學之後有很多分組作業，若是造成其他的組員的麻煩那大概下次就可能沒人會找自己了，在高中若有一些小組討論的時候，就可以盡量積極參與討論，以此來培養團隊合作的能力。

**Comment by lai1008** [comment]
在高中時期，建議多參加各種課外活動和社團，發現自己的興趣所在，且這些經驗對於未來選擇大學科系和職業方向都會有很大幫助。考完學測後也不要荒廢學業，不然銜接到大學的時候會十分辛苦。

**Comment by ShuliANN** [comment]
對高中生來說，獨立思考和培養批判性思維是非常重要的。到了大學，你會發現學習不再是死記硬背，很多問題其實沒有絕對的答案，更多的是依據不同的觀點進行討論和分析。

**Comment by ShuliANN** [comment]
不隨波逐流，保持自己的獨立思考。在學校或生活中，你可能會遇到很多來自同齡人的觀點和看法，但這不代表每個觀點都是對的。學會挑戰常規，思考事情的多面性，了解不同的觀點背後的理由，而不是單純接受「大家都說」的答案。

**Comment by ShuliANN** [comment]
培養問問題的能力，不要害怕提出問題，問題是學習的起點。問問題能讓你深入了解一個主題，並且激發思考。尤其是在大學，很多課程並不會直接給你答案，而是會引導你自己去發現問題和探索答案。

**Comment by Tchang323** [comment]
給高中生的升學建議：對於在比較後半端學校的，我覺得繁星提供一個很好的升學管道機會，雖然在高中三年需要不斷的顧成績，壓力難免比較大，不過你卻不用像其他一樣學測要考很高分才能進到國立頂尖大學，而且也可以提早確定學校，是個不錯的機會。

**Comment by wcyeee** [comment]
我覺得高中可以多多參加跟職涯有關的活動多多嘗試，也順便累積學習歷程，之後申請的時候就不用慌張沒有東西可以放。然後剩下的時間就全部拿來好好讀書，同時也一定要好好照顧自己的身體還有心理，不要讀書讀一讀也把自己的健康讀掉了。

**Comment by forestdew507** [comment]
「盡量可能的探索」這種話我想大家都聽膩了，但這還真的就是最根本的。無論是身為高中生時需要多方探索以確立大學志向，還是大學生需要多方探索以確立畢業後的方向，我覺得不要自我設限、隨時把握可能只有一次的機會，其實都是給人生增添不同驚喜的契機:)

**Comment by qiu0109** [comment]
我覺得高中可以多參加社團活動，學習與人合作和培養做事的能力，到了大學會有更多機會可以辦活動營隊。再來是不需要在高中時就為了成績熬夜苦讀，擁有充足的睡眠是很重要的，睡得好書才會讀得有效率。

**Comment by Mminniehuang** [comment]
我在高中最後悔的就是沒有從高一就開始好好讀書，沒有意識到將來要面對的升學，不論是學測或是很重視在校成績的繁星都一樣。
看到大家逐漸解脫成為大學生，自己卻還要埋頭苦讀分科的日子真的太痛苦了...所以真的要從一開始就好好讀書！

**Comment by yue083139** [comment]
我覺得在高中時期就先確認自己的志向比較重要，常常會看到有些人一進到某些科系就休學重考，真的有點浪費人生的感覺，或是進去之後為了一紙文憑而死撐著做自己不喜歡的事情，也不見得未來做相關就讀科系的行業，再來就是培養規劃時間的能力吧，畢竟上大學要做的事情真的變很多。

**Comment by kai1660** [comment]
可以開始看張旭的微積分課程，多修其他系的課程，曾懭見聞，大學還是要好好努力，不然到之後大三大四想要努力的時候都來不及了，考試名額又這麼少，希望大家都可以好好利用推甄。

**Comment by RoyLin1014** [comment]
社團勇敢玩，但同時也要很認真對待自己的課業，要想辦法兼顧以及找尋課業與社團的完美比例，讓自己的高中生活充實有趣
